gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -113,7 +116,9 @@ def _wait_for_all_operations_done(self):
         request = operations.get(name=operation_name)
         response = request.execute()
         if response['done']:
-          self._handle_failure(operation_name, response)
+          self._handle_failure(
+              self._operation_names_to_test_case_names[operation_name],
```

Comment:
I think it's important to have _both_ the operation name and the test case name. The operation name is the main way in which we can get the logs and details of the run.